errors.ts

```diff
@@ -64,26 +64,38 @@ function resolveCallStack(error: Error): string {
 	return outputMessage;
 }
 
+let attachUncaughtExceptionHandler: any;
 export function installUncaughtExceptionListener(): void {
-	process.on("uncaughtException", (err: Error) => {
-		let callstack = err.stack;
-		if (callstack) {
-			callstack = resolveCallStack(err);
-		}
-		console.error(callstack || err.toString());
+	if(!attachUncaughtExceptionHandler) {
+		attachUncaughtExceptionHandler = _.once(() => {
+			process.on("uncaughtException", (err: Error) => {
+				let callstack = err.stack;
+				if (callstack) {
+					try {
+						callstack = resolveCallStack(err);
+					} catch(err) {
+						console.error("Error while resolving callStack:", err);
+					}
+				}
+				console.error(callstack || err.toString());
+
+				if(!$injector.resolve("staticConfig").disableAnalytics) {
+					try {
+						let analyticsService = $injector.resolve("analyticsService");
```

Comment:
after discussion, we've decided to set `disableAnalytics` to true for Proton, so it common-lib will not track the exceptions. Proton will do it on its own.